### ISSUE TITLE
expected unqualified-id before ‘>’ token

### ISSUE THREAD

**Issue by aaronpeikert** [brandmaier/bnnlib#2]
After running `make all` I get:
```
src/Sequence.cpp:219:37: error: expected unqualified-id before ‘>’ token
  219 |    return std::numeric_limits<double>::quiet_NaN();
      |                                     ^
make[1]: *** [/usr/lib64/R/etc/Makeconf:176: src/Sequence.o] Error 1
make[1]: Leaving directory '/home/aaron/Documents/remote/github/bnnlib'
make: *** [Makefile:9: r] Error 1
```

**Comment by brandmaier** [comment]
Hm. What compiler are you using? It works for me on macOS with the standard C++ compiler. Is this fixed by this: cb6f597f68f4be6c45544d43cfd3cd21b192ffd6
by any chance?

**Comment by aaronpeikert** [comment]
cb6f597 did it for me.
